static/sass/styles.scss

```diff
@@ -1143,3 +1144,95 @@ hr.p-separator.is-shallow {
 .no-border::after {
   content: none !important;
 }
+
+// Custom styles /dell
+$dell-blue: #232943;
+.p-strip--dell-blue {
+  @extend .p-strip--dark;
+
+  background-color: $dell-blue;
+}
+
+.p-card--dell-webinars {
+  background: #000
+    url("https://assets.ubuntu.com/v1/4329b93d-vertical-thumb_111.png") right;
+  color: #fff;
+  height: 400px;
+  margin-top: 1rem;
+  padding: calc(1rem - 1px);
+
+  @media only screen and (max-width: $threshold-6-12-col) {
+    background-color: #000;
+    background-size: cover;
+    height: auto;
+    margin-bottom: 1.5rem;
+  }
+
+  .p-icon--video-play {
+    color: #fff;
+    height: 28px;
+    width: 28px;
+  }
+}
+
+.p-aspect-ratio-image {
+  height: 0;
+  padding-top: calc(200% / 3);
+  width: 100%;
+}
+
+.p-aspect-ratio-image--desktop-solutions {
+  @extend .p-aspect-ratio-image;
+
+  background: #666
+    url("https://assets.ubuntu.com/v1/3c7f0508-DELLBartonGe_3-1661193946868.png")
+    50% no-repeat;
+  background-size: contain;
+  margin-top: 1rem;
+}
+
+.p-aspect-ratio-image--reference-architecture {
+  background: #f7f7f7
+    url("https://assets.ubuntu.com/v1/cb3ad1d2-reference-archtecture-dell (1).png")
+    50% 0 no-repeat;
+  background-size: 100%;
+  margin-top: 1rem;
+  padding-top: calc(130% / 3);
+}
+
+.p-aspect-ratio-image--at-glance {
```

Comment:
we talked about providing 2x3 as a reusable class, these modifiers are quite specific - what if we added 100 images like this - does that mean 100 classes? I suggest putting the reusable stuff on a reusable class, then inline the specific (which is very littlem just the url.)

Reply:
Refactored